setup.cfg

```diff
@@ -103,10 +103,12 @@ commands =
 skip_install = True
 deps =
     black
```

Comment:
What do you think about replacing `black` with `ruff` as well? I don't have experience with ruff's formatting, but it looks like it's designed to be able to replace Black.

Reply:
Haven't had enough experience with it yet, but last I used it it was fairly compliant but the CLI interface was slightly different in a bothersome way. Don't want to do so at this juncture, but definitely something we could align on eventually. Black is generally fast enough not to bother me, but one less tool could be good. I think ruff can handle isort's job too.